src/sentry/web/forms/accounts.py

```diff
@@ -226,7 +226,7 @@ def save(self, commit=True):
 
 class RegistrationForm(PasswordlessRegistrationForm):
     password = forms.CharField(
-        required=True, widget=forms.PasswordInput(attrs={"placeholder": "something super secret"})
+        required=True, widget=forms.PasswordInput(attrs={"placeholder": "something super secret", "id": "registration-password"})
```

Comment:
If we use `auto_id`, we can revert this change:

```suggestion
        required=True, widget=forms.PasswordInput(attrs={"placeholder": "something super secret"})
```